src/msg/async/Event.cc

```diff
@@ -327,10 +339,16 @@ void EventCenter::wakeup()
   ldout(cct, 20) << __func__ << dendl;
   char buf = 'c';
   // wake up "event_wait"
+  #ifdef _WIN32
+  int n = send(notify_send_fd, &buf, sizeof(buf), 0);
+  #else
   int n = write(notify_send_fd, &buf, sizeof(buf));
+  #endif
   if (n < 0) {
-    if (errno != EAGAIN) {
-      ldout(cct, 1) << __func__ << " write notify pipe failed: " << cpp_strerror(errno) << dendl;
+    if (ceph_sock_errno() != EAGAIN)
```

Comment:
no need to change the formatting...

if () {

Reply:
Right, that line used to be significantly longer in previous PR versions. I'll update it.